src/default_to_host.rs

```diff
@@ -0,0 +1,11 @@
+use crate::Triple;
+
+/// A simple wrapper around `Triple` that provides an implementation of
+/// `Default` which defaults to `Triple::host()`.
+pub struct DefaultToHost(pub Triple);
+
+impl Default for DefaultToHost {
+    fn default() -> Self {
+        Self(Triple::host())
+    }
+}
```

Comment:
Maybe move this to `lib.rs` as this is tiny?

Reply:
Done